@cci/cci@v1.ifcx

```diff
@@ -0,0 +1,27 @@
+{
+  "header": {
+    "id": "https://identifier.buildingsmart.org/uri/molio/cciconstruction/1.0",
```

Comment:
I like the idea of using the URIs here. It would be good if that URL could also return IFCX output.